packages/react-component-library/src/components/TabSet/TabSet.tsx

```diff
@@ -42,11 +43,21 @@ export const TabSet: React.FC<TabSetProps | ScrollableTabSetProps> = ({
   isScrollable,
   ...rest
 }) => {
+  const isActiveIndex = Children.toArray(children).findIndex(
+    (item: React.ReactNode) => {
+      if (React.isValidElement(item)) {
+        return item.props.isActive
+      }
+
+      return false
+    }
+  )
+  const initialActiveTab = isActiveIndex === -1 ? 0 : isActiveIndex
```

Comment:
I couldn't call `setActiveTab` when mapping the tab in the render because it triggers 'too many re-renders'.